src/docs/api/design.mdx

```diff
@@ -24,3 +24,54 @@ In the above example we flattened `projects`. The table below shows the existing
 | projects | When the resource is under a specific project in the hierarchy but not under an issue | organizations | {organization_slug}/ {project_slug} | [Create a New Client Key](https://docs.sentry.io/api/projects/create-a-new-client-key/)|
 | issues | When the resource is under a specific issue in the hierarchy | projects | {issue_id} | [List an Issue's Events](https://docs.sentry.io/api/events/list-an-issues-events/)|
 | sentry-app-installations | When the resource is mapped to a specific integration | organizations | {integration_slug} | [Delete an External Issue](https://docs.sentry.io/api/integration/delete-an-external-issue/)|
+
+Here are some additional examples:
+- 👍 `/organizations/{organization_slug}/projects/`
+- 👍 `/projects/{organization_slug}/{project_slug}/issues/`
+- 👎 `/projects/`
+
+**Note:** Hierarchy here does not necessarily mean that one collection belongs to a parent collection. For example `projects/{project_identifier}/teams/` and `teams/{team_identifier}/projects/` can co-exist. The first one means the teams that are working on a specific project and the latter means the projects one specific team is working on.
```

Comment:
Is there a nicer way to show a note?

Reply:
I just pushed a commit that turns this into a Note. It seems like the <Note> component is broken in develop, but this works lol

Reply:
@sentaur-athena - FYI :D